pkg/clients/aws.go

```diff
@@ -67,12 +67,12 @@ func CredentialsIDSecret(data []byte, profile string) (string, string, error) {
 		return "", "", err
 	}
 
-	id, err := iniProfile.GetKey(external.AWSAccessKeyIDEnvVar)
+	id, err := iniProfile.GetKey("AWS_ACCESS_KEY_ID")
```

Comment:
These variables are no longer exported in the v0.23 AWS SDK.

Reply:
I think we can update how this is done as the SDK now supports credentials chain, but I would prefer to defer that to a later PR.

Reply:
Is there an issue tracking this?

Reply:
@muvaf I will tackle it as part of https://github.com/crossplane/provider-aws/issues/148